google-summer-of-code/GSOC-STUDENT-TEMPLATE.md

```diff
@@ -39,8 +43,8 @@
 
 ### About me
 
-> Add your name, personal details and contact information (Slack ID, E-Mail). Optionally, add why you would you consider yourself as the best person for this project.
+> Introduce yourself here in a brief manner. Try to answer basic qustions like Who are you? What are your interest?. Add your name, personal details and contact information (Slack ID, E-Mail).
 
-### Appendix
+### What you expect from the mentor? (optional)
```

Comment:
I think we can still keep it as `Appendix`

Reply:
The appendix heading will also work until students place the right content in this heading.